core/cap-0046-02.md

```diff
@@ -200,6 +200,27 @@ finished, so this can be safely called in the middle of a function.
 fn update_current_contract_wasm(wasm_hash: Bytes)
 ```
 
+### Invoking a contract using `InvokeHostFunctionOp`
+Contracts can be invoked via `InvokeHostFunctionOp` with
+`HOST_FUNCTION_TYPE_INVOKE_CONTRACT` host function type.
+
+The function accepts `InvokeContractArgs` struct that consists of the `contractAddress`, `functionName` and the `args` array.
+
+TODO: describe `auth` requirement.
+
+If the invocation is successful, the return value `ScVal` along with `ContractEvent`s will be 
+used to construct a `InvokeHostFunctionSuccessPreImage`, from which the hash is computed and 
```

Comment:
You should also mention that the full return value and events will be available in the tx meta.